play/json-flattened-view-gen/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "json-flattened-view-gen"
+version = "0.1.0"
+edition = "2018"
+authors = ["Materialize, Inc."]
+license = "proprietary"
```

Comment:
We have usually been OK with erring on the side of giving away non-core tools like this under the apache license, rather than making it proprietary. Cc @rjnn to confirm as a company officer.

Reply:
Yup, feel free to make it apache 2, which is our go-to free license

Reply:
Changed.